### ISSUE TITLE
Accessibility - design: white against [Intermediate] yellow color doesn't meet contrast standards

### ISSUE THREAD

**Issue by eff-soraya** [EFForg/sec#277]
From a quick check with the WAVE accessibility extension on Google Chrome:

![screen shot 2017-11-12 at 7 35 42 pm](https://user-images.githubusercontent.com/16709499/32708747-b34f3ba8-c7e0-11e7-89c6-3b933ed04f44.png)

Could we pick a slightly darker shade that is compliant for the 4.5:1 standard?
Thanks!

**Comment by hughillustration** [comment]
OK, let's adjust our plans somewhat:
Black text
BEGINNING #81BC53
INTERMEDIATE #CCC652
ADVANCED #D9818D
![screen shot 2017-11-15 at 2 08 04 pm](https://user-images.githubusercontent.com/10857936/32862949-a2ab697c-ca0e-11e7-8a07-61cb42109596.png)
